view/js/dropzone-factory.js

```diff
@@ -0,0 +1,65 @@
+var DzFactory = function () {
+	this.createDropzone = function(dropSelector, textareaSelector) {
+		return new Dropzone( dropSelector, {
+			paramName: 'userfile', // The name that will be used to transfer the file
+			maxFilesize: max_imagesize, // MB
+			url: '/media/photo/upload?response=url&album=',
+			addRemoveLinks: true,
+			acceptedFiles: 'image/*',
+			clickable: true,
+			accept: function(file, done) {
+				done();
+			},
+			init: function() {
+				this.on('success', function(file, serverResponse) {
+					const targetTextarea = document.getElementById(textareaSelector);
+					const bbcodeString = $(serverResponse).find('div#content').text();
```

Comment:
Once #12892 is merged, this will have to be changed.

Reply:
Ok.
Also the request-url i think :)

Reply:
Not anymore, #12892 will take care of that.

Reply:
Ah, i understand... it was a bug to show the whole page.